kitsune/sumo/templatetags/jinja_helpers.py

```diff
@@ -113,6 +113,14 @@ def wiki_to_html(wiki_markup, locale=settings.WIKI_DEFAULT_LANGUAGE,
                                              nofollow=nofollow))
 
 
+@library.filter
+def wiki_to_safe_html(wiki_markup, locale=settings.WIKI_DEFAULT_LANGUAGE,
+                      nofollow=True):
+    """Wiki Markup -> HTML jinja2.Markup object with limited tags"""
+    return jinja2.Markup(parser.wiki_to_html(wiki_markup, locale=locale,
+                                             nofollow=nofollow, tags=bleach.ALLOWED_TAGS))
```

Comment:
Maybe we add `<p>` to the allowed tags?. The current editor wraps every paragraph in `<p>`

Reply:
Cool.